src/content/getting-started/quickstart/k3s/_index.md

```diff
@@ -0,0 +1,140 @@
+For connecting clusters with overlapping CIDRs, please refer to the
+[Globalnet guide](../../architecture/globalnet/).
+
+### Prerequisites
+
+1. Prepare two nodes with
+[K3s-supported OS](https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#operating-systems)
+installed (These two nodes are referred to as node-a and node-b).
+2. Decide the Pod CIDR and the Service CIDR for each K3s cluster.
+
+### Deploy K3s Clusters
+
+#### Deploy cluster-a on node-a
+
+In this step you will deploy **cluster-a** on node-a using the the following IP ranges:
+
+| Pod CIDR     | Service CIDR |
+|--------------|--------------|
+|10.44.0.0/16  |10.45.0.0/16  |
+
+To deploy K3s with the IP ranges, run below commands on node-a.
+
+```bash
+POD_CIDR=10.44.0.0/16
+SERVICE_CIDR=10.45.0.0/16
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="--cluster-cidr $POD_CIDR --service-cidr $SERVICE_CIDR" sh -s -
+```
+
+{{% notice tip %}}
+For more information on interacting with K3s, please refer to the [k3s documentation](https://rancher.com/docs/k3s/latest/en/quick-start/).
+{{% /notice %}}
+
+The kubeconfig file needs to be modified like below to make k8s API endpoint point to public IP of the node-a.
+If the IP should be different from the first field of `hostname -I`, modify the value manually.
+
+```bash
+cp /etc/rancher/k3s/k3s.yaml kubeconfig.cluster-a
+IP=$(hostname -I | awk '{print $1}')
+sed -i 's/127.0.0.1/'$IP'/' kubeconfig.cluster-a
+```
+
+#### Deploy cluster-b on node-b
+
+In this step you will deploy **cluster-b** on node-b using the the following IP ranges:
+
+| Pod CIDR     | Service CIDR |
+|--------------|--------------|
+|10.144.0.0/16 |10.145.0.0/16 |
+
+To deploy K3s with the IP ranges, run below commands on node-b.
+
+```bash
+POD_CIDR=10.144.0.0/16
+SERVICE_CIDR=10.145.0.0/16
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="--cluster-cidr $POD_CIDR --service-cidr $SERVICE_CIDR" sh -s -
+```
+
+The kubeconfig file needs to be modified like below to make k8s API endpoint point to public IP of the node-b.
+If the IP should be different from the first field of `hostname -I`, modify the value manually.
+
+```bash
+cp /etc/rancher/k3s/k3s.yaml kubeconfig.cluster-b
+IP=$(hostname -I | awk '{print $1}')
+sed -i 's/127.0.0.1/'$IP'/' kubeconfig.cluster-b
+```
+
+Then, copy kubeconfig.cluster-b to node-a.
+
+#### Install `subctl` on node-a
+
+{{< subctl-install >}}
+
+#### Use cluster-a as Broker
+
+On node-a, run the below command.
+
+```bash
+subctl deploy-broker --kubeconfig kubeconfig.cluster-a --service-discovery
+```
+
+#### Join cluster-a and cluster-b to the Broker
+
+On node-a, run the below commands.
+
+##### Join node-a
+
+```bash
+subctl join --kubeconfig kubeconfig.cluster-a broker-info.subm --clusterid cluster-a --natt=false
+```
+
+##### Join node-b
+
+```bash
+subctl join --kubeconfig kubeconfig.cluster-b broker-info.subm --clusterid cluster-b --natt=false
+```
```

Comment:
I think we can reduce these sections.

#### Join cluster-a to the Broker

```bash
subctl join --kubeconfig kubeconfig.cluster-a broker-info.subm --clusterid cluster-a --natt=false
```

#### Join cluster-b to the Broker

```bash
subctl join --kubeconfig kubeconfig.cluster-b broker-info.subm --clusterid cluster-b --natt=false
```